index.html

```diff
@@ -78,156 +76,501 @@
         <!-- <p>&copy; 2018 Texar. &middot; <a href="#">Privacy</a> &middot; <a href="#">Terms</a> &middot; <a href="http://wang-chenyu.github.io" target="blank"></a></p> -->
         <p>&copy; 2018 Texar.</p>
       </footer>
-
-    </div><!-- /.container -->
-
+    </div>
+    <!-- /.container -->
 
     <!-- Bootstrap core JavaScript
     ================================================== -->
     <!-- copy text js -->
     <script>
       function copyToClipboard(element) {
-        var $temp = $("<input>");
-        $("body").append($temp);
+        var $temp = $('<input>');
+        $('body').append($temp);
         $temp.val($(element).text()).select();
-        document.execCommand("copy");
+        document.execCommand('copy');
         $temp.remove();
       }
     </script>
     <!-- click to change the text on the button -->
     <script>
-      function change(){
-        var lable = $("#change").text().trim();
-         if(lable == "Hide") {
-           $("#glyphicons_halflingsregular").text("Copy");
-         }
-         else {
-           $("#change").text("Copied");
-            }
+      function change() {
+        var lable = $('#change')
+          .text()
+          .trim();
+        if (lable == 'Hide') {
+          $('#glyphicons_halflingsregular').text('Copy');
+        } else {
+          $('#change').text('Copied');
         }
+      }
     </script>
 
     <!-- Placed at the end of the document so the pages load faster -->
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
-    <script>window.jQuery || document.write('<script src="js/vendor/jquery.min.js"><\/script>')</script>
+    <script>
+      window.jQuery ||
+        document.write('<script src="js/vendor/jquery.min.js"><\/script>');
+    </script>
     <script src="js/bootstrap.min.js"></script>
-    <!-- Just to make our placeholder images work. Don't actually copy the next line! -->
-    <script src="js/vendor/holder.min.js"></script>
     <!-- IE10 viewport hack for Surface/desktop Windows 8 bug -->
     <script src="js/ie10-viewport-bug-workaround.js"></script>
     <!--
     <script>$(function(){$(".typed").typed({strings: ["Text Generation Toolkit", "Flexibility", "Modular", "Dialog System", "Machine Translation", "Question Answering"],typeSpeed: 60,loop: true  });});</script>
     -->
     <script src="js/typed.min.js"></script>
+
+    <!-- installer options -->
+    <script>
+      const availableInstallOptions = {
+        'texar-tf,stable,pip,python3.6':
+          'pip install git+https://github.com/asyml/texar#egg=texar',
+        'texar-tf,stable,pip,python3.7':
+          'pip install git+https://github.com/asyml/texar#egg=texar',
+        'texar-pytorch,stable,pip,python3.6':
+          'pip install git+https://github.com/asyml/texar#egg=texar-pytorch',
+        'texar-pytorch,stable,pip,python3.7':
+          'pip install git+https://github.com/asyml/texar#egg=texar-pytorch'
+      };
```

Comment:
add/update this object to specify install command for various combination of `tool/build/package/language`.